app/models/metric/helper.rb

```diff
@@ -165,4 +165,14 @@ def self.get_time_range_from_offset(start_offset, end_offset = nil, options = {}
 
     return start_time, end_time
   end
+
+  def self.get_time_interval(obj, attrs, ts = nil)
+    ts ||= attrs[:timestamp]
+    state = obj.vim_performance_state_for_ts(ts)
+
+    ts = Time.parse(ts).utc if ts.kind_of?(String)
```

Comment:
@chessbyte is their a more ruby way to write `Time.parse(ts).utc if ts.kind_of?(String)` ?